### PR TITLE
Drop python 2 support

### PR DESCRIPTION
**Pull request by jonmmease**
This PR removes the Python 2 circleci tests, updates the minimum Python version in setup.py to 3.6, and adds an f-string to dash.py for good measure.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,12 +53,6 @@ def read_req_file(req_type):
         "Intended Audience :: Science/Research",
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3",
```
Comment (alexcjohnson) [inline setup.py:58]: Is it intentional to drop `Python :: 3`?

Comment (jonmmease) [reply setup.py:58]: ahh, no. good catch

**Comment by alexcjohnson** [comment]
The one other thing I was hoping this PR would do is drop the Py2 parts of `requires-dev.txt` and `requires-testing.txt`

**Review by alexcjohnson — APPROVED** [review]
Love it! 🌅  💃

**Pull request merged**